shared/schemas/quilt_summarize.json

```diff
@@ -44,7 +44,7 @@
 
     "typeShorthand": {
       "type": "string",
-      "enum": ["echarts", "json", "jupyter", "perspective", "vega", "voila"]
+      "enum": ["echarts", "igv", "json", "jupyter", "perspective", "vega", "voila"]
```

Comment:
Isn't `"IGV"` more correct? It's definitely what we should show in the dropdown.

Reply:
Another part of the code is responsible for titles in the dropdown. I fixed "Igv" -> "IGV" there